datasets/conll2003/conll2003.py

```diff
@@ -0,0 +1,133 @@
+# limitations under the License.
+
+# Lint as: python3
+"""Introduction to the CoNLL-2003 Shared Task: Language-Independent Named Entity Recognition"""
+
+import logging
+
+import datasets
+
+
+_CITATION = """\
+@inproceedings{tjong-kim-sang-de-meulder-2003-introduction,
+    title = "Introduction to the {C}o{NLL}-2003 Shared Task: Language-Independent Named Entity Recognition",
+    author = "Tjong Kim Sang, Erik F.  and
+      De Meulder, Fien",
+    booktitle = "Proceedings of the Seventh Conference on Natural Language Learning at {HLT}-{NAACL} 2003",
+    year = "2003",
+    url = "https://www.aclweb.org/anthology/W03-0419",
+    pages = "142--147",
+}
+"""
+
+_DESCRIPTION = """\
+The shared task of CoNLL-2003 concerns language-independent named entity recognition. We will concentrate on
+four types of named entities: persons, locations, organizations and names of miscellaneous entities that do
+not belong to the previous three groups.
+
+The CoNLL-2003 shared task data files contain four columns separated by a single space. Each word has been put on
+a separate line and there is an empty line after each sentence. The first item on each line is a word, the second
+a part-of-speech (POS) tag, the third a syntactic chunk tag and the fourth the named entity tag. The chunk tags
+and the named entity tags have the format I-TYPE which means that the word is inside a phrase of type TYPE. Only
+if two phrases of the same type immediately follow each other, the first word of the second phrase will have tag
+B-TYPE to show that it starts a new phrase. A word with tag O is not part of a phrase.
+
+For more details see https://www.clips.uantwerpen.be/conll2003/ner/ and https://www.aclweb.org/anthology/W03-0419
+"""
+
+_URL = "https://github.com/davidsbatista/NER-datasets/raw/master/CONLL2003/"
+_TRAINING_FILE = "train.txt"
+_DEV_FILE = "valid.txt"
+_TEST_FILE = "test.txt"
+
+
+class Conll2003Config(datasets.BuilderConfig):
+    """BuilderConfig for Conll2003"""
+
+    def __init__(self, **kwargs):
+        """BuilderConfig forConll2003.
+
+        Args:
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(Conll2003Config, self).__init__(**kwargs)
+
+
+class Conll2003(datasets.GeneratorBasedBuilder):
+    """Conll2003 dataset."""
+
+    BUILDER_CONFIGS = [
+        Conll2003Config(name="conll2003", version=datasets.Version("1.0.0"), description="Conll2003 dataset"),
+    ]
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "id": datasets.Value("string"),
+                    "word": datasets.Sequence(datasets.Value("string")),
+                    "pos": datasets.Sequence(datasets.Value("string")),
+                    "chunk": datasets.Sequence(datasets.Value("string")),
+                    "ner": datasets.Sequence(datasets.Value("string")),
+                }
```

Comment:
We have other POS/NER datasets (xtreme/pan-x and Lince)and they name their features "tokens"/"words" and "pos" and "ner"/"ner_tags". I would be cool to have the same names everywhere.

What about using the convention "words" + "pos" + "ner" ?
For CoNNL-2003 there would only be one change: word -> words

Then we could update Lince and xtreme/pan-x

WDYT @thomwolf ?

Reply:
I'm fine with that

Reply:
Ok I'm opening a PR for that